apps/src/templates/sectionProgressV2/InviteToV2ProgressModal.jsx

```diff
@@ -69,30 +69,34 @@ function InviteToV2ProgressModal({
     setInvitationOpen(showInvitation());
   }, [dateProgressTableInvitationDelayed, hasSeenProgressTableInvite]);
 
-  const handleModalClose = React.useCallback(() => {
-    analyticsReporter.sendEvent(EVENTS.PROGRESS_V2_SEEN_INVITATION, {
-      sectionId,
-    });
+  const closeModal = () => {
     setInvitationOpen(false);
-  }, [sectionId]);
+  };
```

Comment:
@kobryan0619 does this still get you the logging you need?

Reply:
It does.